### PR TITLE
Mahdiyeh/Add derivx on header menu

### PR DESCRIPTION
**Pull request by mahdiyeh-deriv**

### CODE REVIEW

File: src/javascript/app/base/header.js [hunk 1]
```diff
@@ -146,6 +149,9 @@ const Header = (() => {
             return;
         }
         const main_domain = `https://app.deriv.${getTopLevelDomain()}`;
+        const is_logged_in = Client.isLoggedIn();
+        const has_dxtrade = (State.getResponse('landing_company.dxtrade_gaming_company') || State.getResponse('landing_company.dxtrade_gaming_company')) && true;
```
Comment (ali-fs) [inline src/javascript/app/base/header.js]:
```suggestion
        const has_dxtrade = !!(State.getResponse('landing_company.dxtrade_gaming_company') || State.getResponse('landing_company.dxtrade_gaming_company'));
```

File: src/javascript/app/base/header.js [hunk 2]
```diff
@@ -169,6 +175,15 @@ const Header = (() => {
                 on_mobile: true,
 
             },
+            ...(should_show_xtrade ? {
+                derivx: {
+                    name     : 'Deriv X',
+                    desc     : 'Trade FX and CFDs on a customisable, easy-to-use trading platform.',
```
Comment (ali-fs) [inline src/javascript/app/base/header.js:183]: Don't we need translation for them? (these and other platforms)

File: src/javascript/app/base/header.js (hunk 1 above)
Comment (mahdiyeh-deriv) [reply src/javascript/app/base/header.js]: fixed

File: src/javascript/app/base/header.js (hunk 2 above)
Comment (mahdiyeh-deriv) [reply src/javascript/app/base/header.js:183]: we didn't have it before, let me check and add a separate card for it if necessary

Comment (ali-fs) [reply src/javascript/app/base/header.js:183]: yes thank you 👍🏻

**Review by ali-fs — DISMISSED** [review]

**Review by balakrishna-deriv — DISMISSED** [review]

**Pull request merged**